File: Java/UberPool.java
package Java;
class UberPool extends Car{
    String brand;
    String model;
    
    public UberPool(Integer id, String license, String driver, Integer passenger, String brand, String model) {
        super(id, license, driver, passenger);
        this.brand = brand;
        this.model = model;
    } 
}

File: PHP/UberPool.php
<?php
    require_once('car.php');
    class UberPool extends Car{
        public $brand;
        public $model;

        public function __construct($license, $driver, $model, $brand) {
            parent::__construct($license, $driver);
            $this->model = $model;
            $this->brand = $brand;
        }
    }
?>